client/extensions/woocommerce/index.js

```diff
@@ -61,6 +62,7 @@ const getStorePages = () => {
 			configKey: 'woocommerce/extension-products',
 			path: '/store/product/:site',
 			parentPath: '/store/products/:site',
+			documentTitle: translate( 'New Product' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **61** times:
`translate( 'Product' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).